html5ever/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "html5ever"
-version = "0.22.5"
+version = "0.23.0"
```

Comment:
I'm pretty sure we don't need a major bump for this change, since we only added dev-dependencies.

Reply:
@jdm Ok, I updated to 22.6, is this ok?

Reply:
Yes.